docs/content/ingestion/firehose.md

```diff
@@ -36,6 +36,28 @@ A sample local firehose spec is shown below:
 |filter|A wildcard filter for files. See [here](http://commons.apache.org/proper/commons-io/apidocs/org/apache/commons/io/filefilter/WildcardFileFilter.html) for more information.|yes|
 |baseDir|directory to search recursively for files to be ingested. |yes|
 
+#### HttpFirehose
+
+This Firehose can be used to read the data from remote sites via HTTP.
+A sample http firehose spec is shown below:
+
+```json
+{
+    "type"    : "http",
+    "uris"  : ["http://example.com/uri1", "http://example2.com/uri2"]
+}
+```
+
+The below configurations can be optionally used for tuning the firehose performance.
+
+|property|description|default|
+|--------|-----------|-------|
+|maxCacheCapacityBytes|Maximum size of the cache space in bytes. 0 means disabling cache.|1073741824|
+|maxFetchCapacityBytes|Maximum size of the fetch space in bytes. 0 means disabling prefetch.|1073741824|
```

Comment:
yeah more documentation could help.

however, as a user I would know how much disk space I am willing to spend on this task and it would be super nice if system automatically decides on the division of how much to use for cache vs prefetch space.
currently user needs to do the tuning that can potentially be done by the system itself.